src/core/feed.factory.ts

```diff
@@ -84,22 +84,29 @@ export class FeedFactory {
     return new BlockedUsersFeed(this.client);
   }
 
-  public directInbox(): DirectInboxFeed {
-    return new DirectInboxFeed(this.client);
+  public directInbox(options: { limit: number; thread_message_limit: number }): DirectInboxFeed {
```

Comment:
These are optional so it's `options: { limit?: number; thread_message_limit?: number } = {}`